install/scripts/hyperdome_client

```diff
@@ -1,9 +1,9 @@
-#!/usr/bin/env python3
```

Comment:
Why are you removing the shebangs? :thinking:

Reply:
easy, I pasted over them